direct/core/src/main/java/cz/o2/proxima/direct/core/DirectDataOperator.java

```diff
@@ -50,8 +50,11 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import java.util.UUID;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.SynchronousQueue;
```

Comment:
unused? how comes spotless did not delete these?

Reply:
Probably before spotless was applied.